File: kattis/listgame2/listgame2.py
x = int(input())
d = 2
count = 0
while d*d <= x:
	if x%d == 0:
		count += 1
	while x%d == 0:
		x //= d
	d += 1

if x > 1:
	count += 1

print(count)

File: kattis/listgame2/listgame2.cpp
#include <iostream>
using namespace std;

typedef unsigned long long ull;

int main(){
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	ull x;
	cin >> x;

	ull count = 0;
	for (ull d = 1; d < x; ++d){
		if (x % d == 0)
			count++;
	}

	cout << count << endl;

	return 0;
}
